src/PaymentRequestHelper.php

```diff
@@ -222,7 +222,11 @@ public static function complement( Payment $payment, AbstractPaymentRequest $req
 		$request->set_additional_data( $additional_data );
 
 		// Metadata.
-		$metadata = [];
+		$metadata = [
+			'network_id' => \get_current_network_id(),
+			'blog_id'    => \get_current_blog_id(),
+			'payment_id' => $payment->get_id(),
```

Comment:
Should we prefix these with e.g. `wp_` or `wordpress_` to make it clear where this metadata relates to?

Reply:
I don't think this is necessary:

<img width="666" alt="Schermafbeelding 2022-05-19 om 15 32 47" src="https://user-images.githubusercontent.com/869674/169305659-9f9136a7-ba6e-4faf-b39e-ba69cb9c9eca.png">